### ISSUE TITLE
Enhancement: Introduce back the job to create environments on demand

### ISSUE THREAD

**Issue by deors** [Accenture/adop-cartridge-java#14]
Before the introduction of environment type DEV|PROD, the environment job allowed for creating as many environments as needed on demand, and then the deploy job could be used to deploy any build to any environment as needed.
This has been so far a very useful feature in demos and presentations, and one useful for those cases in which the project is not mature enough to go directly to production, but need to have feature environments to test and validate specific stuff before merging and deploying to production.
Could we bring back the ability to create new environments on demand with any desired name, and then manually control via deployment what features are put in them?

**Comment by nickdgriffin** [comment]
Yes, it's on my list of random things that need doing - happy for a PR to go in for it too. I think it should be creating just a "NAME" environment for the dev type and "NAME_A" and "NAME_B" environments for the prod type.
